pymc3/stats.py

```diff
@@ -216,7 +220,7 @@ def waic(trace, model=None, pointwise=False, progressbar=False):
 
     vars_lpd = np.var(log_py, axis=0)
     if np.any(vars_lpd > 0.4):
-        warnings.warn("""For one or more samples the posterior variance of the
+        _log.warning("""For one or more samples the posterior variance of the
```

Comment:
Really not sure here. We might also want to change this to `backend.report.SamplerWarning`. The functionally should match, only the name seems a bit off...
Which maybe indicates that the name `SamplerWarning` isn't right.